site/app/Helpers/Helpers.php

```diff
@@ -154,35 +154,25 @@ public static function fileStatusBadge(string $status): string
     }
 
     /**
-     * Get file url for given filename
-     */
-    public static function fileUrl(string $filename): string
-    {
-        return asset('storage/uploads/files/'.$filename);
-    }
-
-    /**
-     * Check whether the file is processed and ready
+     * Check whether the file has a specific status
+     *
+     * @param  string  $status (App\Enums\FileStatus)
      */
-    public static function isFileReady(File $file): bool
+    public static function isFileStatus(File $file, string $status): bool
```

Comment:
isFileStatus(File $file, FileStatus $status):

Reply:
`isFileStatus(File $file, FileStatus $status)` -> 
`App\Helpers\Helpers::isFileStatus(): Argument #2 ($status) must be of type App\Enums\FileStatus, string given`

Il faudrait refactor pour en faire de véritables enums.